packages/patternlab/styleguide/source/_patterns/02-molecules/collapsible-header.json

```diff
@@ -2,6 +2,7 @@
   "collapsibleHeader": {
     "info": "Click here to read more about this thing",
     "title": "Collapsible Header",
+    "state": "false",

```

Comment:
can you add this new prop in the component markdown doc?

Reply:
I think we should make it a boolean, to be consistent. What do you think?

Reply:
Yes, that makes sense.  The state is added as boolean to the MD.  Thank you for mentioning that.